### ISSUE TITLE
Jacoco-Comparison-Tool Integration with Jenkins

### ISSUE THREAD

**Issue by AmrutaM** [mariamr/Jacoco-Comparison-Tool#2]
Is there a way we can use this tool in integration to Jenkins or sonar?

**Comment by mariamr** [comment]
Yes, I don't see any reason you couldn't. Jenkins can run shell scripts. You can configure Jenkins to run the report.sh file provided in the project, and that will take care of generating the reports

**Comment by AmrutaM** [comment]
That comparison will not be sufficient. I need something which can also
tell me who has broke the code. With Comparison i can get only difference
of 2 builds results.

On Wed, Aug 26, 2015 at 1:28 PM, Mariam Rajabi notifications@github.com
wrote:

> Yes, I don't see any reason you couldn't. Jenkins can run shell scripts.
> You can configure Jenkins to run the report.sh file provided in the
> project, and that will take care of generating the reports
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mariamr/Jacoco-Comparison-Tool/issues/2#issuecomment-134889106
> .

## 

Thanks,
Amruta Mistry.
9099197380
